Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fedora:31
+FROM fedora:32
```

Comment:
Fedora 32 goes EOL at the end of this month. Fedora 33 is supported through October, so maybe choose that.

Reply:
Done